src/thunderbird_accounts/mail/tests/test_tasks.py

```diff
@@ -193,31 +193,6 @@ def test_success_account_already_exists_on_stalwart(self):
             self.assertEqual(oidc_id, task_results.get('oidc_id'))
             self.assertEqual(mock_stalwart_pkid, task_results.get('stalwart_pkid'))
 
-    def test_account_already_exists(self):
```

Comment:
We now update the account in place, this fixes an issue we're experiencing where people who have an account already but haven't subscribed (so like everyone at mzla) are currently not having their account references created.